docs/AOTA.md

```diff
@@ -0,0 +1,43 @@
+# AOTA Updates
+
+## Supported AOTA commands and their functionality:
+
+### Supported ***docker*** commands
+
+| *docker* Command | Definition                                                                                                                                                                                                                                       |
+|:-----------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Import           | Importing an image to the device and starting a container                                                                                                                                                                                        |
+| List             | Lists all containers for all images that are either 'latest' or have a tag number.  It will list the container ID, state, and image name.  It will provide 'NONE' for the container ID and state if the image does not have an active container. | 
+| Load             | Loading an image from the device and starting a container                                                                                                                                                                                        |
+| Pull             | Pulls an image or a repository from a registry and starting a container                                                                                                                                                                          |
```

Comment:
What does it mean to pull a repository from a docker registry?

Reply:
@gblewis1 - This is all moved from the other documents and was the same in all of them.  I can try and rephrase it and hopefully it makes sense.  Do you have any suggestions?

Reply:
@gblewis1 - I didn't read these and just moved.  The grammar is really bad, so I'll try and fix that on all of them.  Unfortunately, these issues are all over our documents.

Reply:
@gblewis1 - The pull description matches what the docker documentation also says.  https://docs.docker.com/engine/reference/commandline/pull/ Fixed the grammar on the rest of the descriptions.